receiver/singletonreceivercreator/README.md

```diff
@@ -39,8 +39,8 @@ The configuration consists of two parts:
 ### Leader Election Configuration
 | configuration       | description                                                                     | default value      |
 |---------------------|---------------------------------------------------------------------------------|--------------------|
-| **lease_name**      | The name of the lease object.                                                   | singleton-receiver |
-| **lease_namespace** | The namespace of the lease object.                                              | default            |
+| **lease_name**      | The name of the lease object.                                                   | none (required)    |
```

Comment:
does this mean that this value is required, but we don't ship it with a default? So users must set a value, otherwise it's broken?

Reply:
Yes, exactly

Reply:
Do we prompt the user to enter a value or do they have to figure it out on their own? :D

Reply:
I mean, it's a custom OTel Collector component; there is no way to prompt a user. But we have a validation rule, the user will see it in the logs. Furthermore, it's not meant to be used by users directly; it's used by Telemetry Manager, where we actually make sure that the lease name and namespace are set.